### PR TITLE
new files

### PR DESCRIPTION
**Pull request by SaadMahi**

### CODE REVIEW

**Comment by fayez-baig** [comment]
@SaadMahi Always give meaningful branch name

and your commit-msg should be what you have done in 2-3 words in a short explanation kind off

**Comment by SaadMahi** [comment]
> @SaadMahi Always give meaningful branch name
> 
> and your commit-msg should be what you have done in 2-3 words in a short explanation kind off

will keep in mind

File: loading.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style-1.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (murtazshaikh) [inline loading.html]: title ?

Comment (murtazshaikh) [reply loading.html]: it makes the project look attractive

**Review by murtazshaikh — APPROVED** [review]

File: loading.html (hunk 1 above)
Comment (SaadMahi) [reply loading.html]: title updated

**Pull request merged**